stravalib/client.py

```diff
@@ -31,6 +31,20 @@
 from stravalib.unithelper import is_quantity_type
 from stravalib.util import limiter
 
+VALID_STREAM_TYPE = [
+    "time",
+    "latlng",
+    "distance",
+    "altitude",
+    "velocity_smooth",
+    "heartrate",
+    "cadence",
+    "watts",
+    "temp",
+    "moving",
+    "grade_smooth",
+]
+
```

Comment:
We can use `strava_model.StreamType.schema()['enum']` to get these values from the OpenAPI spec.

Reply:
Oh that's really nice.